ui/shared/status/StatusToolTip.qml

```diff
@@ -5,7 +5,8 @@ import "../../shared"
 
 ToolTip {
     id: tooltip
-    implicitWidth: textContent.implicitWidth + Style.current.bigPadding
+    property int maxWidth: 400
```

Comment:
The `maxWidth` of 400 is only for the emoji reactions. In other cases, we don't need it. So here, you can put something very big and in the emoji reaction tooltip set `maxWidth: 400`

Reply:
Done